.github/workflows/release-nuget.yml

```diff
@@ -0,0 +1,34 @@
+# Workflow creating a Github package
+name: Release Nuget
+
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Verify commit exists in origin/main
+      run: |
+        git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
+        git branch --remote --contains | grep origin/main
+    - name: Set VERSION variable from tag
+      run: echo "VERSION=${GITHUB_REF/refs\/tags\/v/}" >> $GITHUB_ENV
+    - name: Set SOLUTION variable to point to solution file
+      run: |
+        SOLUTION=$(find . -name '*.sln' -printf "%p" -quit)
+        echo "SOLUTION=${SOLUTION}" >> $GITHUB_ENV
+    - name: Build
+      run: dotnet build --configuration Release /p:Version=${VERSION} ${SOLUTION}
+    - name: Test
+      run: dotnet test --configuration Release /p:Version=${VERSION} --no-build ${SOLUTION}
+    - name: Pack
+      run: dotnet pack --configuration Release /p:Version=${VERSION} --no-build --output . ${SOLUTION}
```

Comment:
Would be great if we could include debug symbols when creating the package.

Reply:
That does sound useful. You're thinking like a separate .snupkg file? I'll have to read up on how that works with Github packages (multiple files).

Reply:
Yep. Was thinkin .snupkg file. https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg